website/docs/onboarding/containers/creating-a-lima-instance-with-podman-desktop.md

```diff
@@ -20,79 +20,52 @@ Consider creating a custom Lima instance to:
 #### Prerequisites
 
 - The `limactl` executable is installed.
+  See [Installing Lima](https://lima-vm.io/docs/installation/).
 
-```shell
-brew install lima
-```
-
-See <https://lima-vm.io>
+  ```shell-session
+  $ brew install lima
+  ```
 
 #### Procedure
 
-Currently you need to use the console, to create a new Lima instance.
-
-After the instance is started, you need to restart the Lima extension.
+1. In a terminal, create the Lima machine.
 
-Examples:
+   - To create a Lima machine with rootless Podman, use the `podman` template:
 
-To create an instance "podman" from a template "podman":
+     ```shell-session
+     $ limactl start --name=podman template://podman
+     ```
 
-```console
-$ limactl start --name=podman template://podman
-```
+   - To create a Lima machine with rootful Podman, use the `podman-rootful` template:
 
-To create an instance "docker" from a template "docker":
+     ```shell-session
+     $ limactl start --name=podman template://podman-rootful
+     ```
 
-```console
-$ limactl start --name=docker template://docker
-```
+   - To create an Lima machine with rootless Docker, use the `docker` template:
 
-To select the number of CPUs and the memory or disk size:
+     ```shell-session
+     $ limactl start --name=docker template://docker
+     ```
 
```

Comment:
To create a Lima machine with rootful Docker, use the `docker-rootful` template:
     ```shell-session
     $ limactl start --name=podman template://docker-rootful
     ```

Reply:
added